app/ports/migrations/0012_add_10.15_worker.py

```diff
@@ -0,0 +1,17 @@
+from django.db import migrations
+
+
+def add_new_builder(apps, schema_editor):
+    Builder = apps.get_model('ports', 'Builder')
+    Builder.objects.create(name='10.15_x86_64', display_name='10.15')
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ports', '0011_populate_display_name.py'),
```

Comment:
This migration did not get applied. After some testing I found that adding `.py` to the last migration's name causes it to fail, so I believe it to be the most probable reason.

We can try to fix this by removing this `.py` but I am spectacle about it, as it did not work for my development setup and I had to make a new migration.

Reply:
@neverpanic must have received an email regarding the failure.

Reply:
Haven't noticed. Maybe I missed it, though.